src/libsystemd-network/ndisc-internal.h

```diff
@@ -47,6 +47,15 @@ struct sd_ndisc {
         void *userdata;
 };
 
+#define SD_RADV_DEFAULT_MIN_TIMEOUT             200 * USEC_PER_SEC
+#define SD_RADV_DEFAULT_MAX_TIMEOUT             600 * USEC_PER_SEC
+
+#define SD_RADV_MAX_INITIAL_RTR_ADVERT_INTERVAL 16 * USEC_PER_SEC
+#define SD_RADV_MAX_INITIAL_RTR_ADVERTISEMENTS  3
+#define SD_RADV_MAX_FINAL_RTR_ADVERTISEMENTS    3
+#define SD_RADV_MIN_DELAY_BETWEEN_RAS           3
+#define SD_RADV_MAX_RA_DELAY_TIME               500 * MSEC_PER_SEC
```

Comment:
also, I'd prefer if we'd stick to one time unit only everywhere, and so far that was usecs, and not msecs.

also, I think it would be good to clarify the time unit by suffixing macros like this with `_USEC` or so